test/12_nullablearray.jl

```diff
@@ -1,16 +1,13 @@
 module TestNullableArray
-
 using Base.Test
 using CategoricalArrays
 using CategoricalArrays: DefaultRefType, catvaluetype, leveltype
 
 const ≅ = isequal
 
-for ordered in (false, true)
-    for R in (CategoricalArrays.DefaultRefType, UInt8, UInt, Int8, Int)
-        # Vector with no null values
```

Comment:
That comment included an interesting information, opposed to "vector with nulls values" below.